doc/features/address-resolution/README.md

```diff
@@ -0,0 +1,66 @@
+# Address resolution
+
+The driver auto-detects new Cassandra nodes when they are added to the cluster by means of server-side push
+notifications and checking the system tables.
+
+For each node, the address the driver receives the address set as `rpc_address` in the node's cassandra.yaml file. In
```

Comment:
Should we mention the nuance about `broadcast_rpc_address` here in C* 2.1+, or is that probably too much info?